Documentation/Storage-Configuration/Advanced/ceph-osd-mgmt.md

```diff
@@ -190,3 +190,29 @@ If you don't see a new OSD automatically created, restart the operator (by delet
 
 !!! note
     The OSD might have a different ID than the previous OSD that was replaced.
+
+
+## OSD Migration
+
+Ceph does not support changing certain settings on existing OSDs. To support changing these settings on an OSD, the OSD must be destroyed and re-created with the new settings. Rook will automate this by migrating only one OSD at a time. The operator waits for the data to rebalance (pgs to become `active+clean`) before migrating the next OSD. This ensures that there is no data loss. Refer [osd-migration](https://github.com/rook/rook/blob/master/design/ceph/osd-migration.md) design doc for more information. 
+
+Following scenarios are supported for OSD migration:
+
```

Comment:
Let's have a bullet list where we add the scenarios with a simple one-line description. Then we just need one example. A bullet such as:
```
- Enable or disable OSD encryption for PVC-based OSDs by change the `encrypted` setting under the `storageClassDeviceSets`
```

Reply:
Got a bit confused about this suggestion. Updated the PR. Let me know if that's what you were suggesting.

Reply:
Yes, that's what I was thinking, thanks. Now when we support other scenarios we can add to this list.